doc/sphinx/users-guide/vcl-built-in-subs.rst

```diff
@@ -417,6 +421,10 @@ The `vcl_backend_response` subroutine may terminate with calling
     lookups hitting this object will be turned into passed transactions,
     as if ``vcl_recv`` had returned ``pass``.
 
+    ``error(status code, reason)``
+    Abandon the backend request and control will be passed to
```

Comment:
I'm not sure we want to use the word "Abandon" here, it's confusing since we already have `return (abandon)`. We can use the same wording as `synth` instead.

Reply:
Got it, check the new force push